package/run.sh

```diff
@@ -9,12 +9,25 @@ while [ "$STACK_NAME" == "" ]; do
   sleep 1
   STACK_NAME=$(wget -q -O - ${META_URL}/self/stack/name)
 done
+# Get etcd service certificates
+UUID=$(curl -s http://rancher-metadata/2015-12-19/stacks/Kubernetes/services/etcd/uuid)
+ACTION=$(curl -s -u $CATTLE_ACCESS_KEY:$CATTLE_SECRET_KEY "$CATTLE_URL/services?uuid=$UUID" | jq -r '.data[0].actions.certificate')
+
+if [ -n "$ACTION" ]; then
+    mkdir -p /etc/etcd/ssl
+    cd /etc/etcd/ssl
+    curl -s -u $CATTLE_ACCESS_KEY:$CATTLE_SECRET_KEY -X POST $ACTION > certs.zip
```

Comment:
i think we should either do a retry/count or exit when we can't pull the certificate

Reply:
We wait for metadata, if we can't get certs by then, the container will restart on it's own.